src/lamp_py/runtime_utils/process_logger.py

```diff
@@ -61,8 +70,10 @@ def add_metadata(self, **metadata: MdValues) -> None:
             if key in ProcessLogger.protected_keys:
                 continue
             self.metadata[str(key)] = str(value)
-        self.default_data["status"] = "add_metadata"
-        logging.info(self._get_log_string())
+
+        if self.default_data.get("status") is not None:
+            self.default_data["status"] = "add_metadata"
+            logging.info(self._get_log_string())
```

Comment:
is this something we'll want on all the time?

Reply:
I put this in the stop log printing before `log_start()` gets called, was kind of confusing for me to see logs in splunk before the log had actually started, but I can add that back if you'd prefer.